Differential Equations.tex

```diff
@@ -284,10 +284,13 @@ \subsubsection*{Solution}
 \item Plug in the ICs to get $a_n, b_n$.
 
 \begin{gather*}
-a_n = \frac{1}{l} \int_{-l}^{l} f(x) \sin(\frac{n\pi}{l}x) dx \\
+a_n = \frac{1}{l} \int_{-l}^{l} f(x) \cos(\frac{n\pi}{l}x) dx \\
```

Comment:
@JaKXz Plugging in `u(x,0)=f(x)`, you get `T(0) = a_n*1 + b_n*0` and `X(x)=sin(n*pi/x)`. So `f(x) = sum[a_n sin(n*pi/x)]` and so the a_n integral is a `sine` one.